app/controllers/admin/boxes_controller.rb

```diff
@@ -52,6 +52,6 @@ def set_box
   end
 
   def box_params
-    params.require(:box).permit(:name, :uri, :short_name, :color)
+    params.require(:box).permit(:api_connection_id, :name, :uri, :short_name, :color, settings: {})
```

Comment:
Toto je potencialna diera ak tam niekto podhodi vymyslene id, treba aj validaciu na urovni modelu, ze to je connection_id z tenanta

Reply:
Okay, pridala som.